index.html

```diff
@@ -103,25 +121,44 @@ <h2>PORTFOLIO</h2>
 			</div>
 			
 		</section>
-		
 	</main>
+
 	<footer class="contact" id="contact">
 		<div class= contact-wrapper>
 			<h2>CONTACT ME</h2>
-		<p>The best way to get in touch with me is via email, but if you prefer social media I have included those too.</p>
-		<div class="contact-icons-wrapper">
+			<p>The best way to get in touch with me is via email, but if you prefer social media or the contact form I have included those too.</p>
+		
+			<div class="contact-icons-wrapper">
 
-			<!--i tags to insert icons of each social media link-->
-			<a class="icon" href="https:linkedin.com/in/albertoroldanq" target="_blank"><i class="fab fa-linkedin"></i></a>
-			<a class="icon" href="mailto:albertoroldanq@gmail.com" target="_blank"><i class="fas fa-envelope"></i></a>
-			<a class="icon" href="https://github.com/Albertoroldanq" target="_blank"><i class="fab fa-github-square"></i></a>
-		</div>
+				<!--i tags to insert icons of each social media link-->
+				<a class="icon" href="https:linkedin.com/in/albertoroldanq" target="_blank">
+					<i class="fab fa-linkedin"></i>
+				</a>
+				<a class="icon" href="mailto:albertoroldanq@gmail.com" target="_blank">
+					<i class="fas fa-envelope"></i>
+				</a>
+				<a class="icon" href="https://github.com/Albertoroldanq" target="_blank">
+					<i class="fab fa-github-square"></i>
+				</a>
+
+			</div>
 		</div>
 
+
 		<!--
-			POSSIBLE CONTACT FORM FROM GITHUB PAGES
+			POSSIBLE CONTACT FORM FOR GITHUB PAGES
 			https://dev.to/charalambosioannou/create-a-static-webpage-with-a-contact-form-on-github-pages-3532
 		-->
+
+			<div class="contact-form">
+				<iframe name="hiddenFrame" class="hide"></iframe>
+                <form action="https://formspree.io/f/xayadevj" method="POST" target="hiddenFrame">
```

Comment:
not sure if you're needing more indents here or if you did this for a reason? Since the form is inside the div might make sense to add some indents?

Reply:
Totally! It should be fixed now, thanks!